src/app/_components/RecentBlocks/NewBlockPlaceholder.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Icon, Stack } from '@chakra-ui/react';
+import { ArrowsClockwise } from '@phosphor-icons/react';
+import { useEffect, useRef, useState } from 'react';
+
+import { Text } from '../../../ui/Text';
+import { useSubscribeBlocks } from '../BlockList/Sockets/useSubscribeBlocks';
+import { BTC_BLOCK_MIN_WIDTH, EMPTY_BTC_BLOCK_WIDTH } from './consts';
+
+export function NewBlockPlaceholder({
+  newestBtcBlockHeight,
+  refetch,
+}: {
+  newestBtcBlockHeight: number;
+  refetch: () => void;
+}) {
+  const [hasNewBlocks, setHasNewBlocks] = useState(false);
+  const lastBlockHeightRef = useRef(newestBtcBlockHeight);
+  useSubscribeBlocks(true, block => {
```

Comment:
With true hardcoded it seems like this socket connect and block subscription never turns off, but I thought we wanted to turn it off after receiving one block